Gemfile.lock

```diff
@@ -1,161 +1,179 @@
+GIT
+  remote: https://github.com/thoughtbot/shoulda-matchers.git
+  revision: 4b160bd19ecca7f97d7ac22dccd5fde9b0da5a9f
+  branch: rails-5
+  specs:
+    shoulda-matchers (3.1.2)
+      activesupport (>= 4.2.0)
+
 PATH
   remote: .
   specs:
     bsm_oa (0.3.1)
-      bsm-models
-      doorkeeper (~> 3.0.0)
+      bsm-models (~> 0.9)
```

Comment:
is that a dependency we can drop altogether? I doubt we really need/use it

Reply:
so it does use bsm-models and that is currently at`(>= 5.0, < 5.1)` I'lll update that one now.